images/plbase/plbase-install.sh

```diff
@@ -54,7 +54,11 @@ dnf -y install postgresql15-server-devel
 cd /tmp
 git clone --branch v0.7.0 https://github.com/pgvector/pgvector.git
 cd pgvector
-make
+# This Docker image will be built in GitHub Actions but must run on a variety
+# of platforms, so we need to build it without machine-specific instructions.
+# https://github.com/pgvector/pgvector/issues/143
+# https://github.com/pgvector/pgvector/issues/143
```

Comment:
Did you mean to include the same issue URL twice?

Reply:
Nope! Good catch. Resolved in d63b9a5.